draft-tigress-requirements.md

```diff
@@ -112,7 +112,7 @@ General terms:
 # Assumptions
 
 - Original credential (with cryptographic key material) MUST NOT be sent or shared. Instead, sender SHALL be transferring its approval token for Receiver to acquire a new credential.
-- Credential Authority SHALL NOT not allow for two users to use the same credential / cryptographic keys.
+- Credential Authority SHALL NOT allow for two users to use the same credential / cryptographic keys.
```

Comment:
Credential Authority was not defined, did we mean to say, "Provisioning Partner"?